admin-dev/themes/new-theme/js/pages/order/create.js

```diff
@@ -50,10 +50,21 @@ function refreshAddressesList(refreshCartAddresses) {
   }
 }
 
+/**
+ * proxy to allow other scripts within the Create Order page to refresh cart
+ */
+function refreshCart() {
+  if (orderPageManager === null) {
+    console.log('Error: Could not refresh addresses list as orderPageManager is null');
```

Comment:
Hello,

Is that the best way to handle this error ? Not sure, how do we do on other pages?

Reply:
I think, it's not enough, I'm still getting it also on when the form in the model is submitted with invalid data: 
![image](https://user-images.githubusercontent.com/32385991/76748875-358d0600-6784-11ea-8554-4d4d5fdaf344.png)

Reply:
Though, it's not critical error as I noticed only design breaks a little near the `save` button

Reply:
I don't understand 😆, is your answer really linked with what I asked? Am I dumb? 😸 

It looks like your js error is not linked at all with the console log above

Reply:
Btw is that mergeable then?

Reply:
Ofcourse you are not. @Progi1984  requested a code refactoring. Code refactoring does not solve the error and gives better code quality. As a more backend developer, it would take me quite long to figure out these Javascripts so I asked your help on this one. Anyway, thank you for giving some attention :)

Reply:
My answer was not linked for what you asked